From 5667ae2e83166dedca55a4bd4252f812322780b4 Mon Sep 17 00:00:00 2001
From: Maarten Lankhorst <wine@mblankhorst.nl>
Date: Sat, 28 Sep 2013 10:00:37 +0200
Subject: [PATCH 39/40] Revert "winmm: Increase mmdevapi buffer sizes."

This reverts commit fdfff7cf6013a030f2b240b13b02831e502d49e7.

You've got to be kidding me, 1 second latency in winmm because
of a driver not used on linux? If the 'upstream' audio devs
spent less time on workarounds and more on fixing drivers,
this kind of crap wouldn't be needed.

I cannot believe that winecoreaudio requires .5 second latency,
their api allows for very low latency, so it's probably
winecoreaudio screwing things up instead.
---
 dlls/winmm/waveform.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dlls/winmm/waveform.c b/dlls/winmm/waveform.c
index 23adfc0..97849ff 100644
--- a/dlls/winmm/waveform.c
+++ b/dlls/winmm/waveform.c
@@ -68,8 +68,8 @@ static const WCHAR muteW[] = {'M','u','t','e',0};
  *   - We must be able to identify bad devices without crashing.
  */
 
-/* buffer size = 100 * 100000 (100 ns) = 1 second */
-#define AC_BUFLEN (100 * 100000)
+/* buffer size = 10 * 100000 (100 ns) = 0.1 seconds */
+#define AC_BUFLEN (10 * 100000)
 #define MAX_DEVICES 256
 #define MAPPER_INDEX 0x3F
 
-- 
1.8.3.2

